# try:
#     print(2/0)
# except:
#     print("wrong output.")

# try:
#     print(glitch)
# except:
#     print("There's nothing")
# else:
#     print("A value has passed.")
    
# glitch = input("FIX")
# print(glitch)
